packages/sources/coinpaprika-test/src/endpoint/crypto-single.ts

```diff
@@ -0,0 +1,48 @@
+import { PriceEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { RestTransport } from '@chainlink/external-adapter-framework/transports'
+import { DEFAULT_API_ENDPOINT, PRO_API_ENDPOINT } from '../config'
+import { buildUrlPath, EndpointTypesSingle, inputParameters } from '../crypto-utils'
+
+const restEndpointTransport = new RestTransport<EndpointTypesSingle>({
```

Comment:
v2 had support for this endpoint, but since coinpaprika has a batched one we should drop support for this one (that is, just delete it from the v3 EA)